R/utils_gcp.R

```diff
@@ -0,0 +1,105 @@
+# Semi-automated tests of Google Cloud Storage integration live in tests/gcp/. # nolint
+# These tests should not be fully automated because they
+# automatically create S3 buckets and upload data,
+# which could put an unexpected and unfair burden on
+# external contributors from the open source community.
+# nocov start
+gcp_gcs_exists <- function(key,
```

Comment:
As picky as this sounds, would you give each argument its own indented line and the closing parenthesis / opening brace a line of their own?

```r
cp_gcs_exists <- function(
  key,
   bucket = gcp_gcs_bucket(),
   version = NULL
) {
```